src/main/java/openblocks/Config.java

```diff
@@ -380,6 +380,10 @@
 	@ConfigProperty(category = "devnull", name = "sneakClickToGui", comment = "If true, /dev/null will require sneaking in addition to clicking air to open gui")
 	public static boolean devNullSneakGui = true;
 
+	@OnLineModifiable
+	@ConfigProperty(category = "hangglider", name = "enableThermal", comment = "Enable a whole new level of hanggliding experience through thermal lift. See keybindings for acoustic vario controls")
+	public static boolean hanggliderEnableThermal = true;
```

Comment:
I'm thinking about setting this to 'false' by default in 1.7.10. Next release will be probably last for this version, so I don't want to make significant gameplay changes this late.

Though this should definitely be enabled in later versions - I was already wondering what can be done to make it viable alternative to elytra wings.